.github/workflows/promote-to-stable.yml

```diff
@@ -0,0 +1,58 @@
+name: Promote
+
+on:
+  issue_comment:
+    types:
+      - created
+
+permissions:
+  issues: write
+
+jobs:
+  promote:
+    name: ⬆️ Promote to stable
+    environment: "2204 Branch"
+    runs-on: ubuntu-latest
+    if: |
+      ( !github.event.issue.pull_request )
+      && contains(github.event.comment.body, '/promote ')
+      && contains(github.event.*.labels.*.name, 'testing')
+    steps:
+      - name: ⬆️ Promote to stable
+        uses: snapcrafters/ci/promote-to-stable@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          store-token: ${{ secrets.SNAP_STORE_STABLE }}
+          snapcraft-project-root: "ffmpeg-2204-sdk"
+
+  update-content-snap:
+    name: Update the content snap
+    needs: [ release-sdk, get-architectures ]
+    environment: "2204 Branch"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update the content snap
+        uses: snapcrafters/ci/sync-version@main
+        with:
+          token: ${{ secrets.SNAPCRAFTERS_BOT_COMMIT }}
+          snapcraft-project-root: "ffmpeg-2204"
+          update-script: |
+            sed -i 's/^\(- ffmpeg-2204-sdk\/latest\/\)\(.*\)$/\1stable/' ffmpeg-2204/snapcraft.yaml
```

Comment:
Don't change the version during promotion. Only change it during release to candidate.

Reply:
Is it impossible to create a release on promotion? This would be the best possible way IMO, else, we'll always point to a temporary branch of ffmpeg-sdk snap.